pkgs/tools/misc/ufetch-nixos/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "ufetch-nixos";
+  src = ./ufetch-nixos;
+
+  phases = "installPhase";
+
+  installPhase = ''
+    mkdir -p $out
```

Comment:
The script should be installed to `$out/bin` so that installing the package adds it on path.

You can use [`install`](https://linux.die.net/man/1/install) command `install -D $src  $out/bin/ufetch`, which will create the directory at the same time.